extensions/wikia/AdEngine/js/lookup/prebid/adapters/beachfront.js

```diff
@@ -0,0 +1,65 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.beachfront', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.context.slotsContext',
+	'ext.wikia.aRecoveryEngine.instartLogic.recovery',
+	'wikia.log'
+], function (adContext, slotsContext, instartLogic, log) {
+	'use strict';
+
+	var bidderName = 'beachfront',
+		bidfloor = 0.01,
```

Comment:
Can we confirm this floor with @nefretete or Lincoln? I believe we set global floors much higher for outstreem.

Reply:
Maybe we should move it to wikiFactory to allow revOps Team adjusting it when required?

Reply:
Confirmed by @nefretete: hardcode to 0.01. Also additional question was sent to Beachfront.